sorc/grid_tools.fd/filter_topo.fd/filter_topo.F90

```diff
@@ -1144,6 +1147,42 @@ subroutine fill_cubic_grid_halo(data, data2, halo, ioff, joff, sign1, sign2)
 
   end subroutine fill_cubic_grid_halo
 
+  !> This routine extrapolate geolat_c and geolon_c halo points for the
+  !! regional standalone grid. Halo points are needed for dxc and dyc
+  !! calculation.
+  !!
+  !! @param[in] data  ???
+  !! @param[in] halo ???
+  !! @author Ratko
+  subroutine fill_regional_halo(data, halo)
```

Comment:
We are required to do unit tests for all new routines. These tests run under github actions. I can help you write the test. Just ask.

Reply:
I'm adding descriptions. I see everywhere @parm[in], does doxygen recognize [inout]?

Reply:
Yes, I will need help to create test.

Reply:
> I'm adding descriptions. I see everywhere @parm[in], does doxygen recognize [inout]?

Yes.

Reply:
> > I'm adding descriptions. I see everywhere @parm[in], does doxygen recognize [inout]?
> 
> Yes.

Oh, I see in manual, syntax is [in,out].